src/charts/charts.less

```diff
@@ -11,6 +11,7 @@ pf-c3-chart {
     width: 100%;
   }
 }
+
```

Comment:
Looks like this file's changes could be discarded. No point updating to include an empty line 😉